qa/logging-config/src/test/java/org/elasticsearch/common/logging/JsonLoggerTests.java

```diff
@@ -100,18 +102,17 @@ public void testDeprecatedMessageWithoutXOpaqueId() throws IOException {
 
             assertThat(jsonLogs, contains(
                 allOf(
-                    hasEntry("type", "deprecation"),
+                    hasEntry("event.dataset", "deprecation"),
                     hasEntry("log.level", "DEPRECATION"),
                     hasEntry("log.logger", "deprecation.test"),
-                    hasEntry("cluster.name", "elasticsearch"),
-                    hasEntry("node.name", "sample-name"),
+                    hasEntry("elasticsearch.cluster.name", "elasticsearch"),
+                    hasEntry("elasticsearch.node.name", "sample-name"),
                     hasEntry("message", "deprecated message1"),
                     hasEntry("data_stream.type", "logs"),
-                    hasEntry("data_stream.dataset", "deprecation.elasticsearch"),
-                    hasEntry("data_stream.namespace", "default"),
+                    hasEntry("data_stream.dataset", "elasticsearch.deprecation"),
                     hasEntry("ecs.version", DeprecatedMessage.ECS_VERSION),
-                    hasEntry("key", "a key"),
-                    not(hasKey("x-opaque-id"))
+                    hasEntry(KEY_FIELD_NAME, "a key"),
```

Comment:
maybe we could use DeprecatedMessage.KEY_FIELD_NAME instead of static import?
I like static imports when context is clear - like static imports of hamcrest matchers in tests, but most of the time I feel like classname.field is better

Reply:
I've qualified the imports as you suggest.